### ISSUE TITLE
exports vs export

### ISSUE THREAD

**Issue by chrisguttandin** [devongovett/unplugin-parcel-macros#10]
I noticed that both examples use the `exports` syntax. I was wondering if that is a requirement?

https://github.com/devongovett/unplugin-parcel-macros/blob/d523d9f7c768050e83e5bedc503df75377a3d68a/examples/next/macro.js#L1

https://github.com/devongovett/unplugin-parcel-macros/blob/d523d9f7c768050e83e5bedc503df75377a3d68a/examples/vite/macro.js#L1

When changing the syntax to EcmaScript modules (`export const css`) I get the following error:

```txt
[plugin:unplugin-macros] × Error loading macro: Unexpected token 'export'
   ╭─[test.js:1:1]
 1 │ import './style.css'
 2 │ import { readFileSync } from 'fs' with {type: 'macro'};
 3 │ import { setupCounter } from './counter.js'
 4 │ import { css } from "./macro" with {type:'macro'};
   · ──────────────────────────────────────────────────
 5 │ 
 6 │ document.querySelector('#app').innerHTML = `
 7 │   <div>
   ╰────
```

Are EcmaScript modules supported? Do I need to change the configuration to enable ESM support?

**Comment by devongovett** [comment]
Yeah. It uses node's module loader so you need to either use the .mjs extension or add "type": "module" to your package.json.

**Comment by chrisguttandin** [comment]
Thanks a lot for the quick clarification. In hindsight it's embarrassingly obvious.
